acceptance_tests/features/pages/reporting_unit.py

```diff
@@ -6,7 +6,7 @@
 
 def go_to(ru_ref):
     browser.visit(f"{Config.RESPONSE_OPERATIONS_UI}/reporting-units/{ru_ref}")
-    time.sleep(0.1)
+    time.sleep(1)
```

Comment:
Should be waiting for an element on the page instead of an arbitrary wait (I know you didn't add it initially) otherwise this will lead to flaky tests

Reply:
@benjefferies  this test is already flaky that s why I have increased the time. I agree that WebDriverWait would be a better solution but we are talking baout existing code. Can we pleas eopen a tech deb card about it, and leave this oustise this code review ?

Reply:
https://trello.com/c/n9Q7lPaL/85-acceptance-tests-should-wait-for-elements-to-appear-instead-of-arbitrary-waits-sleeps